GoogleForCreators/web-stories-wp
# Design System: Quaternary button

## Context

Design added a fourth button type to avoid the custom "Create new Story" button on the side nav of the dashboard 😎 

## Summary

Introducing `BUTTON_TYPES.QUATERNARY`, fourth button color variant. 

## Relevant Technical Choices

Added quaternary specific background and borders. There's overlap but this way if this button style ever changes, it's not tied to something with other purposes. 
Had to override the focus specifically in the `quaternaryColors`, `common` tries to inject its default box-shadow  on anchors - which is what the "button" on the dashboard this is getting used for is, so found it early. Not an issue for other buttons because of how borders work for those vs. this one.

## To-do



## User-facing changes

Dashboard "Create new Story" button styles have changed
| New | Old |
| --- | --- |
| ![quaternary](https://user-images.githubusercontent.com/10720454/111234335-8dcf3180-85ab-11eb-8cb7-fdd758f7d371.gif) | ![old new story](https://user-images.githubusercontent.com/10720454/111234376-a5a6b580-85ab-11eb-9604-d43fffc357f5.gif) |


## Testing Instructions

- See the quaternary button added to storybook, should look like the one found in[ figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3438%3A125786)

- And also look to see the new quaternary button in action on dashboard

### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?

No

### Does this PR change what data or activity we track or use?

No

### Does this PR have a legal-related impact?

No

## Checklist




---



Fixes #6758
